.github/workflows/build.yml

```diff
@@ -594,7 +594,7 @@ jobs:
       (github.event_name == 'push' && github.ref == 'refs/heads/master')
     runs-on:
       labels: bare-metal
-    needs: ["set-tags", "build"]
+    needs: ["set-tags", "build", "dev-test"]
```

Comment:
tracing test should be able to run even if dev-tests didn't run yet

Reply:
This is also a experimentation, the pipeline sometimes fails unexpectedly and I think it may be related to running multiple services simultaneously. Cannot prove yet, but this should give us a confirmation.

All builds after this change have been consistent.